### PR TITLE
PMM-9190 Add Integration tests for Webhook Channel for IA

### PR DESCRIPTION
**Pull request by ElenaFedichkina**
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests-matrix/detail/pmm2-ui-tests-matrix/424/pipeline

### CODE REVIEW

**Review by peterSirotnak — DISMISSED** [review]
Looks good to me.

File: tests/ia/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+* PAGER_DUTY_API_KEY
+### Command to execute Integrated Alerting Alerts tests command
+
+`npx codeceptjs run -c pr.codecept.js tests/ia/alerts_test.js --steps`
+
+### Command to execute all Integrated Alerting tests:
+
+`npx codeceptjs run -c pr.codecept.js --grep '@ia'`
+
+### Execute Integrated Alerting tests in Jenkins
```
Comment (puneet0191) [inline tests/ia/README.md:18]: lovely

File: tests/ia/alerts_test.js [hunk 2]
```diff
@@ -57,20 +61,47 @@ BeforeSuite(async ({
     testEmail,
   );
 
-  ruleIdForEmailCheck = await rulesAPI.createAlertRule({
+  // Preparation steps for checking Alert via webhook server
+  // eslint-disable-next-line no-template-curly-in-string
+  await I.verifyCommand('bash -x ${PWD}/testdata/ia/gencerts.sh');
+  await I.verifyCommand('docker-compose -f docker-compose-webhook.yml up -d');
```
Comment (puneet0191) [inline tests/ia/alerts_test.js:67]: Do you need a wait right after starting the container?

File: tests/ia/pages/api/alertsAPI.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 const { I } = inject();
 const assert = require('assert');
+const { api } = require('@pagerduty/pdjs');
```
Comment (puneet0191) [inline tests/ia/pages/api/alertsAPI.js:3]: the naming API? confuses me

File: tests/ia/README.md (hunk 1 above)
Comment (puneet0191) [reply tests/ia/README.md:18]: may be setting up webhook on local, not using the test setup, just instructions on how many compose I need to start

**Review by puneet0191 — DISMISSED** [review]
Good stuff! thank you a lot

File: tests/ia/README.md (hunk 1 above)
Comment (yurkovychv) [reply tests/ia/README.md:18]: webhook start on local is done automatically from the BeforeSuite hook. So no need to do anything from user perspective

File: tests/ia/alerts_test.js (hunk 2 above)
Comment (yurkovychv) [reply tests/ia/alerts_test.js:67]: there is no need to wait as we have like 2 mins before the alert will be triggered

File: tests/ia/pages/api/alertsAPI.js (hunk 3 above)
Comment (yurkovychv) [reply tests/ia/pages/api/alertsAPI.js:3]: this is the name of an object from the library. This is how they call it. https://github.com/PagerDuty/pdjs#rest-api

File: tests/ia/README.md (hunk 1 above)
Comment (puneet0191) [reply tests/ia/README.md:18]: I meant to say, if I just needed the setup, without having to run the test, could I use it somehow, my objective is to get a ready environment with webhook via staging, so I could pass it like a flag, or add the same steps with framework

Comment (yurkovychv) [reply tests/ia/README.md:34]: done

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**